pom.xml

```diff
@@ -4593,8 +4593,9 @@
                 together, so we just exclude it from pitest check. -->
                 <param>destroy</param>
               </excludedMethods>
-              <coverageThreshold>100</coverageThreshold>
-              <mutationThreshold>97</mutationThreshold>
+              <!-- Temporary reduction until https://github.com/checkstyle/checkstyle/pull/11231 gets merged -->
+              <coverageThreshold>98</coverageThreshold>
+              <mutationThreshold>94</mutationThreshold>
```

Comment:
I think it is ok, such numbers have less role after we introduced survival reports.